ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -88,19 +98,13 @@ chain:
         --annotations=hypershift.openshift.io/skip-release-image-validation=true \
         --additional-tags="expirationDate=$(date -d '4 hours' --iso=minutes --utc)"
 
-      # The timeout should be much lower, this is due to https://bugzilla.redhat.com/show_bug.cgi?id=2060091
-      echo "Waiting for cluster to become available"
-      oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${CLUSTER_NAME}
+      until [[ "$(oc get -n clusters hostedcluster/${CLUSTER_NAME} -o jsonpath='{.status.version.history[?(@.state!="")].state}')" = Completed ]]; do
+          sleep 15
+      done
+      echo "Hosted cluster ${CLUSTER_NAME} completes progressing"
```

Comment:
nit: seems redundant given the echo below

Reply:
Done.